app/views/proposals/new.html.slim

```diff
@@ -0,0 +1,11 @@
+h1 Propose a Deputy Director
+= render partial: 'shared/flash_messages'
+= form_tag pq_proposal_path do |f|
+  fieldset
```

Comment:
I think fieldset tag is not needed since it's just one field without a legend

Reply:
Removed fieldset